src/main/python/__init__.py

```diff
@@ -151,14 +151,15 @@ def _ppr_single_target(self, edge_type, seeds, target, relative_error=0.1, reset
 
 
     # TempestDB methods
-    def ppr(self, edge_type, seeds, num_steps=100000, reset_probability=0.3, max_results = None):
+    def ppr(self, edge_type, seeds, seed_node_type, target_node_type, num_steps=100000, reset_probability=0.3, alternating = True, max_results = None):
```

Comment:
Can you make a note here that alternating = True is needed for directed graphs? (And explain what it does)?